### ISSUE TITLE
Change output directory?

### ISSUE THREAD

**Issue by hab9sr** [lkdjiin/coco#61]
There doesn't seem to be a way to specify the output directory for reports. This would be helpful.

**Comment by lkdjiin** [comment]
This should be an easy option to add and I will be more than happy to add it if only I could see a use case for it.

I'm very curious to know what's wrong with `coverage` and which other directory name do you need?

**Comment by hab9sr** [comment]
`coverage/` is fine for the name. I want to move it one level (say, to a docs/ directory outside the root of the source code)

**Comment by lkdjiin** [comment]
Yes, it should be possible to add this option. Let me take a closer look.
In the meantime could you tell me more on *why* you would need this ? (I never had to do this, and so it's seems weird to me, I just would like to understand)

**Comment by hab9sr** [comment]
Working on a project in which all the documentation (coverage, test plans, etc.) needs to stay in a separate folder from the source code.

Structure should be something like this

\<project root>
- docs/
  - coverage/
  - ...
- app/
  - app/
  - config/
  - ...

**Comment by lkdjiin** [comment]
Ok, thanks for the explanation. I think I'll be able to give it a try tomorrow

**Comment by lkdjiin** [comment]
Hello @hab9sr 
It's done in the branch https://github.com/lkdjiin/coco/tree/coverage-directory
There is a new option `output_directory`. You can set it as follow for your specific need:

    :output_directory: "../docs/coverage"

Could you give it a try and then tell me if it's ok for you?

**Comment by hab9sr** [comment]
Perfect; that's exactly what I needed. Thanks!
